sources/packages/web/src/views/aest/institution/Restrictions.vue

```diff
@@ -1,79 +1,76 @@
 <template>
-  <v-card class="mt-4">
-    <div class="mx-5 py-4">
-      <content-group>
-        <v-row class="m-2">
-          <v-col class="category-header-medium color-blue"
-            >All Restrictions</v-col
-          >
-          <v-col
-            ><v-btn
-              @click="addInstitutionRestriction"
-              class="float-right primary-btn-background"
-              ><font-awesome-icon :icon="['fas', 'plus']" class="mr-2" />Add
-              restriction</v-btn
-            ></v-col
-          >
-        </v-row>
-        <DataTable
-          :value="institutionRestrictions"
-          :paginator="true"
-          :rows="DEFAULT_PAGE_LIMIT"
-          :rowsPerPageOptions="PAGINATION_LIST"
+  <full-page-container
+    :layout-template="LayoutTemplates.CenteredCard"
+    :full-width="true"
+  >
+    <body-header title="All Restrictions" class="m-1">
+      <template #actions
+        ><v-btn
+          @click="addInstitutionRestriction"
+          class="float-right"
+          color="primary"
+          prepend-icon="fa:fa fa-plus-circle"
+          >Add restriction</v-btn
+        ></template
+      >
+    </body-header>
+    <content-group>
+      <DataTable
+        :value="institutionRestrictions"
+        :paginator="true"
+        :rows="DEFAULT_PAGE_LIMIT"
+        :rowsPerPageOptions="PAGINATION_LIST"
+      >
+        <template #empty>
+          <p class="text-center font-weight-bold">No records found.</p>
+        </template>
+        <Column
+          field="restrictionCategory"
+          header="Category"
+          sortable="true"
+        ></Column>
+        <Column field="description" header="Reason">
+          <template #body="slotProps">{{
+            `${slotProps.data.restrictionCode} - ${slotProps.data.description}`
+          }}</template></Column
         >
-          <template #empty>
-            <p class="text-center font-weight-bold">No records found.</p>
+        <Column field="createdAt" header="Added"
+          ><template #body="slotProps">{{
+            dateOnlyLongString(slotProps.data.createdAt)
+          }}</template></Column
+        >
+        <Column field="updatedAt" header="Resolved">
+          <template #body="slotProps">{{
+            slotProps.data.isActive
+              ? "-"
+              : dateOnlyLongString(slotProps.data.updatedAt)
+          }}</template></Column
+        >
+        <Column field="isActive" header="Status">
+          <template #body="slotProps">
+            <status-chip-restriction
+              :status="
+                slotProps.data.isActive
```

Comment:
Is this transformation needed? Since the `status-chip-restriction` is specific to the restriction can it be directly receiving the `RestrictionStatus`?

Reply:
here `isActive` is a boolean field. I believe the transformation logic is added to translate the boolean value to restriction status